teamsphisher.py

```diff
@@ -386,6 +389,32 @@ def createThread(skypeToken, senderInfo, targetInfo):
 
     return threadID
 
+def removeExternalUser(skypeToken, senderInfo, threadID, targetInfo):
+    headers = {
+        "Authentication": "skypetoken=" + skypeToken,
+        "User-Agent": useragent,
+        "Content-Type": "application/json",
+        "Origin": "https://teams.microsoft.com",
+        "Referer": "https://teams.microsoft.com/"
+    }
+
+    # Get the current thread information
+    response = requests.get(f"https://amer.ng.msg.teams.microsoft.com/v1/threads/{threadID}", headers=headers)
+    if response.status_code != 200:
+        p_warn("Error retrieving thread information: %d" % (response.status_code))
+        return None
+
+    thread = response.json()
+
+    # Delete the target user from the thread
+    content = requests.delete(f"https://amer.ng.msg.teams.microsoft.com/v1/threads/{threadID}/members/{targetInfo.get('mri')}", headers=headers)
+    print(content.text)
+    if content.status_code != 204:
```

Comment:
In my testing I was receiving a 200 OK back, not a 204; request you change this line to:

```
    if content.status_code != 204 and content.status_code != 200:
```

Reply:
Updated the pull request with these changes